test/e2e/storage/persistent_volumes-local.go

```diff
@@ -988,7 +988,8 @@ func makeLocalPodWithNodeName(config *localTestConfig, volume *localTestVolume,
 	if pod == nil {
 		return
 	}
-	pod.Spec.NodeName = nodeName
+
+	e2epod.SetNodeAffinity(pod, nodeName)
```

Comment:
not gonna use the fancy `NodeSelection` stuff here?

Reply:
not at the moment. The local storage tests are specific to single nodes.